### PR TITLE
Git repo patterns need single-quotes

### PR DESCRIPTION
**Pull request by jogjayr**
See https://github.com/spring-cloud/spring-cloud-config/issues/727

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@jogjayr Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=728)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=728) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by jogjayr** [comment]
I have signed the CLA now

File: docs/src/main/asciidoc/spring-cloud-config.adoc [hunk 1]
```diff
@@ -197,10 +197,10 @@ spring:
           repos:
             simple: https://github.com/simple/config-repo
             special:
-              pattern: special*/dev*,*special*/dev*
+              pattern: 'special*/dev*,*special*/dev*'
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-config.adoc]: I believe single quotes are only needed if the value starts with asterisk.

**Comment by pivotal-issuemaster** [comment]
@jogjayr Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-config&pullRequestId=728)!

**Comment by jogjayr** [comment]
@spencergibb done. do you need me to squash my changes into one commit also?

File: docs/src/main/asciidoc/spring-cloud-config.adoc (hunk 1 above)
Comment (jogjayr) [inline docs/src/main/asciidoc/spring-cloud-config.adoc]: Done

**Comment by ryanjbaxter** [comment]
@jogjayr would you mind submitting the PR against the 1.3.x branch so we can easily merge it forward into master and the 2.0.x branch?

**Comment by jogjayr** [comment]
@ryanjbaxter certainly. I'll close this PR and open a new one for it.

**Comment by jogjayr** [comment]
https://github.com/spring-cloud/spring-cloud-config/pull/729

**Pull request closed without merging**